doc/admin-guide/plugins/cache_range_requests.en.rst

```diff
@@ -112,37 +112,6 @@ which includes information about the partial content range.  In this mode,
 all requests (include partial content) will use consistent hashing method
 for parent selection.
 
-X-Crr-Ims header support
```

Comment:
are there any guidelines on removing parameters or configs? (i.e. configs are being deprecated notification) or when users upgrade their ATS version and their configs are not longer valid, they are expected to check ATS docs of their replacement?

Reply:
I'll put some thought into that.  I had originally tagged this for ats11 but notice it was bumped back to 10.2.